bin/cfg2asm

```diff
@@ -2,9 +2,12 @@
 
 require 'seafoam'
 
-ARGV.each do |file|
-  parser = Seafoam::CFG::CFGParser.new(file)
-  parser.skip_over_cfg 'After code installation'
-  nmethod = parser.read_nmethod
-  p nmethod
-end
+file, verbosity = ARGV
```

Comment:
What if the user writes the wrong number of arguments? Also I'd recommend you covert verbosity to an integer, and check its range, here rather than later, so all argument handling code is in one place.

You'll need some kind of `--help` support too - look at what the other commands do.